packages/client/src/lib/animation/elementAnimations.ts

```diff
@@ -17,28 +17,27 @@ This function will return a Timeline containing all the animations.
 */
 
 type Element = any;
-export type FlipOptions = {
+export type MoveOptions = {
   duration?: number;
   stagger?: number;
   ease?: number;
   onComplete?: gsap.Callback;
   delay?: number;
 };
 
-export function flip(
+export function move(
   elements: Element[],
-  changeFunc: () => void,
-  { onComplete, delay = 0, ...vars }: FlipOptions = {},
+  updateFunc: () => void,
+  { onComplete, delay = 0, ...vars }: MoveOptions = {},
 ) {
   const gsapElements = gsap.utils.toArray(elements);
   const tl = gsap.timeline({ onComplete, delay });
   const bounds = gsapElements.map((el: Element) => el.getBoundingClientRect());
   const copy: any = {};
-  gsapElements.forEach((el: Element) => {
```

Comment:
why do you want to remove this? Couldn't it be useful for example for the undo function?

Reply:
If it is useful, we can add it back then. Also it might require some more work anyway, to send some undo/revert flag here, play back the chained animations back in reverse order, etc
But for the undo, we would also have to decide if we want the animation to always finish (while the undo button is disabled) and then be able to undo. Which might be easier.

Reply:
Ok